### PR TITLE
Full v1 direct to Kafka with Single Topic

### PR DESCRIPTION
**Pull request by larrybabb**
Addresses bulk of #4 
Here's the first pass which does all the work of receiving, validating the incoming release github webhook payload for releases. It then looks for and parses the 3 tsv files that we've contracted with MONDO to produce for each release. It then publishes a json record for each record in those respective TSV files to a configured pubsub topic.

NOTE: I need to add a read.me with the config settings for the GCP function environment variables still.

### CODE REVIEW

**Review by sjahl — COMMENTED** [review]
Overall approach here looks fine to me -- I think we had discussed prior to the break that we'd have the function publish directly to the desired kafka stream, so I'll wait for those changes and can do another review then.

File: main.py
Comment (theferrit32) [inline main.py]: One suggestion is to return an http status code on these error cases. I think it defaults to `200` if none is given. In flask you can return a tuple like `return ("msg", 403)`. I think it just ends up going to the log, but might be useful for filtering log messages. I think it would also flag it on the github side in the webhook history page.

Comment (larrybabb) [reply main.py]: done. updated PR, please re-review

**Comment by larrybabb** [comment]
removed pubsub approach and refactored to send 3 event types to a single kafka/confluent prod topic called 'mondo-notifier' directly.

File: main.py
Comment (sjahl) [inline main.py]: It looks like `KafkaError` is imported but not used

File: main.py
Comment (sjahl) [inline main.py]: You've got a mix of 4 space and 2 space indents -- please use 4 spaces for all indents. Python will get sad if someone changes this later and uses 4 space indents in the wrong spot 🐍

File: main.py
Comment (sjahl) [inline main.py:129]: the `msg` parameter here only gets used in a statement that's commented out

Comment (sjahl) [reply main.py:129]: If the msg parameter is there to meet the `on_delivery` api, there might be a more pythonic way (not sure what that is though... `**kwargs`?) to ignore params that we're not going to use

**Review by sjahl — CHANGES_REQUESTED** [review]
Made some comments inline, main things I'd like to see changed here are:

- Formatting. Biggest issue with this for me is the inconsistent indentation throughout, which is a maintainability problem down the line. Pylint should help surface these formatting issues, and we should probably run it as a github action on PRs
- There's a ton of print statements in here for debugging. I would either remove them, or make them log messages that can be optionally shown with a loglevel.

File: main.py
Comment (larrybabb) [reply main.py]: fixed.

File: main.py
Comment (larrybabb) [reply main.py]: done

File: main.py
Comment (larrybabb) [reply main.py:129]: done

**Review by sjahl — APPROVED** [review]
Thanks Larry, this looks ok to me

**Pull request merged**
